integration/config/core.yaml

```diff
@@ -559,44 +576,28 @@ chaincode:
     # whitelist, add "myscc: enable" to the list below, and register in
     # chaincode/importsysccs.go
     system:
+        _lifecycle: enable
         cscc: enable
         lscc: enable
         escc: enable
         vscc: enable
         qscc: enable
-        tlcc: enable  # FPC Addition:4
-
-    # System chaincode plugins:
-    # System chaincodes can be loaded as shared objects compiled as Go plugins.
-    # See examples/plugins/scc for an example.
-    # Plugins must be white listed in the chaincode.system section above.
-    systemPlugins:
-      # example configuration:
-      # - enabled: true
-      #   name: myscc
-      #   path: /opt/lib/myscc.so
-      #   invokableExternal: true
-      #   invokableCC2CC: true
-      #
-      # FPC Addition:5  
-      - enabled: true
-        name: tlcc
-        path: ../../tlcc/tlcc.so
-        invokableExternal: true
-        invokableCC2CC: true
+        #tlcc: enable  # FPC Addition:4
 
     # Logging section for the chaincode container
     logging:
       # Default level for all loggers within the chaincode container
+      #level:  info
       level:  debug
       # Override default level for the 'shim' logger
-      shim:   info
+      #shim:   warning
+      shim:  info 
```

Comment:
I am wondering why this is still in the config as shim.logger was removed?

Reply:
good question. The stanza is definitely still there in the `fabric-samples.git` as it is in `sampleconfig/core.yaml` of `fabric.git` ....

Reply:
So by default that option is in the new config by default